torch/_torch_docs.py

```diff
@@ -1916,6 +1922,8 @@ def merge_dicts(*dicts):
     tensor([-1.7120,  0.1734, -0.0478, -0.0922])
     >>> torch.clamp(a, min=-0.5, max=0.5)
     tensor([-0.5000,  0.1734, -0.0478, -0.0922])
+    >>> torch.clamp(a, min=8.0, max=11.0)
```

Comment:
Should there be an example for this special case or is the note sufficient?

Reply:
I think a note alone makes sense because this is a degenerate case we don't want to highlight with an example

Reply:
I removed this with the last commit.